tests/std/tests/P2494R2_move_only_range_adaptors/test.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <iterator>
+#include <ranges>
+#include <stdio.h>

```

Comment:
`<stdio.h>` is unused (and if it were used, we're now trying to consistently include `<cmeow>`).